samples/sensor/mcp9600/README.rst

```diff
@@ -0,0 +1,43 @@
+.. _mcp9600:
+
+mcp9600 Temperature Sensor
+###################################
+
+Overview
+********
+
+This sample application periodically reads temperature data from the first
```

Comment:
Please add a link to manufacturer information about this sensor.

Reply:
ok

Reply:
added